src/api/1/query-syntax/index.md

```diff
@@ -0,0 +1,99 @@
+
+### Body encoding
+
+Body contents can be sent in the following formats:
+
+* `application/json`: raw JSON
+* `multipart/form-data`: HTML forms; both field-value pairs and field-files pairs can be sent that way
+
+If a HTML form is sent that way, the resulting body content will be translated into a JSON object, with as many keys as the provided form fields.  
+If the form field holds a file, then the corresponding JSON key will refer to an object instead of a mere value, with the following properties:
+
+  * `filename`: file's name
+  * `encoding`: file encoding
+  * `mimetype`: MIME type
+  * `file`: file content, encoded in base64
+
+---
+
+## Other protocols
+
+<div class="alert alert-info">Kuzzle's extensible protocol system allows communication in virtually any format. This documentation section describes the format that must be used to pass queries to Kuzzle itself, either directly by users (for instance, our <code>MQTT</code> protocol), or indirectly, translated by the custom protocol layer.</div>
+
+Queries made to Kuzzle must be encoded using JSON, and have the following format:
+
+```javascript
+{
+  // required by all queries
+  "controller": "<controller>",
+  "action": "<action>",
+
+  // optional, can be added to all queries
+  "requestId": "<unique request identifier>",
+  "jwt": "<token>",
+
+  // commonly found parameters
+  "index": "<index>",
+  "collection": "<collection>",
+  "body": {
+    // query content
+  },
+  "_id": "<unique ID>"
+}
+```
+
+### Required parameters:
+
+The following 2 parameters are required by all API requests, as these are directly used by Kuzzle to redirect the query to the correct API action:
+
+* `controller`: accessed Kuzzle API controller 
+* `action`: API controller action to be executed
+
+Depending on the API route executed, other parameters may be required. Those are detailed in the corresponding API sections.
+
+### Commonly found parameters:
+
+There are 2 parameters that can be provided to all queries, independently to the API route executed:
+
+* `jwt`: user's authentification token, obtained through the [login]({{ site_base_path }}api/1/controller-aut/login) method
+* `requestId`: user-defined request identifier. Kuzzle does not guarantee that responses are sent back in the same order than queries are made; use that field to link responses to their  query of origin
+* `volatile`: user-defined data, without any impact to the query. Use that object to pass information about the query itself to real-time subscribers. Read more [here]({{site_base_path}}/api/1/volatile-data)
+
+Additionally, a few other parameters are very commonly found in API queries:
+
+* `_id`: unique identifier (e.g. document ID, user kuid, memory storage key, ...)
+* `body`: query content (e.g. document content, message content, mappings, ...)
+* `collection`: data collection
+* `index`: data index
```

Comment:
Maybe we can add `refresh` ?

Reply:
Compared to the total number of API routes, `refresh` is not so common: by nature it can only be used with routes writing into Elasticsearch. Moreover, the necessary explanation for that one is quite length, I don't think it brings anything to add it here.